packages/common/src/dbt/schemas/lightdashMetadata.json

```diff
@@ -1,5 +1,24 @@
 {
     "definitions": {
+        "RequiredAttributes": {
```

Comment:
Define reusable type + improve YML validation to make sure the values are strings or array of strings.